features/telemetry/protocols/flows/pom.xml

```diff
@@ -42,6 +42,17 @@
       <artifactId>org.opennms.features.telemetry.protocols.common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
This module should only depend on the Flow API (`org.opennms.features.flows.api`) and not have direct knowledge of the implementation.